cluster/gce/config-common.sh

```diff
@@ -159,3 +159,7 @@ export WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-k
 export WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
 # Pause container image for Windows container.
 export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.2.1"
+# Storage Path for csi-proxy
+export CSI_PROXY_STORAGE_PATH="https://storage.googleapis.com/gke-release-staging/csi-proxy"
```

Comment:
Why pulling from gke-release-staging?

Reply:
currently it is only available in staging. So this is temporary. Once it is in release, we will update it.

Reply:
Is there a presubmit that can test this change?

Reply:
you mean windows tests by triggering /test pull-kubernetes-e2e-windows-gce?

Reply:
it is available in gke-release now, right?

Reply:
no, not yet.